xml/ay_bigfile.xml

```diff
@@ -390,23 +381,22 @@ exit 0
      be treated as reserved words in the default namespace.
     </para>
     <para>
-     Global attributes are defined in the configuration namespace and must
```

Comment:
BTW I wondered why this is talking about "global" attributes, because I thought there is no such thing as local attributes.

Then I found that all the tables which document the individual **properties** of the various resources are using the heading **Attribute**. IMHO we should globally replace that with **Property** so that we don't create a confusion with the established XML term.